src/main/java/edu/wpi/first/pathweaver/ObservableArrayList.java

```diff
@@ -0,0 +1,45 @@
+package edu.wpi.first.pathweaver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javafx.collections.ModifiableObservableListBase;
+
+public class ObservableArrayList<T> extends ModifiableObservableListBase<T> {
```

Comment:
Why not use https://docs.oracle.com/javase/8/javafx/api/javafx/collections/FXCollections.html#observableArrayList--?

Reply:
Must have missed that while trying to find implementing versions of ObservableList. Fixed.